### ISSUE TITLE
Throws "Cannot redeclare block-scoped variable 'ngDevMode'" in angular 6

### ISSUE THREAD

**Issue by RezaT1994** [qligier/angular4-events#4]
I just updated my angular to 6 and now i have this error:

![image](https://user-images.githubusercontent.com/31609638/39753787-3c6b2fda-52d5-11e8-9012-9b93108663ca.png)

When in remove angular4-events from ng.modules.ts, it works fine.

**Comment by qligier** [comment]
Hey,
Can you try with the new version (`npm install --save angular4-events@5.0.1`)? It should fix issues with Angular 6 but I still haven't had time to upgrade my own projects, so feedback would be appreciated.

**Comment by boehlerlukas** [comment]
@qligier thank's for this repo! Its a great lib! Just tested to upgrade to 5.0.1 but the error still occurs.

**Comment by boehlerlukas** [comment]
@qligier it seems like buid 5.0.0 did work for me, but 5.0.1 didn't. (sorry was on angular5)

**Comment by qligier** [comment]
Yep, version 5.0.1 is only compatible with Angular 6, I should have named it v6.0.0. The README has been updated to clarify that.

**Comment by qligier** [comment]
Closing this issue as it should have been fixed with v5.0.1 update.

**Comment by OlehRb** [comment]
@qligier Hi there, I am having the same issue, but using 5.0.1 doesn't help. I tried to install your library through yarn and npm, but none of the attempts finished successfully.
My **angular** version is 6.0.3, **angular4-events** version is 5.0.1.

Is there something else I can do about it or am I missing something?

**Comment by OlehRb** [comment]
@qligier, hmm, I guess it is issue with **yarn**, because initially I installed some of modules through it and they worked fine, but your module even on 5.0.1 still gave the same error. Unfortunately installing your module through **npm** somehow removed some modules, which were previously installed using yarn. Finally, I dropped _node_modules/_ folder, did **npm i** and I don't get this error anymore.

It would be awesome if you check  why installing of your module on version 5.0.1 using **yarn** fails.
Also, thanks for an awesome library)
